src/logic/basic.lean

```diff
@@ -74,6 +74,8 @@ assume ⟨h⟩, h.elim
 -- missing [symm] attribute for ne in core.
 attribute [symm] ne.symm
 
+lemma ne.symm_iff {α} {a b : α} : a ≠ b ↔ b ≠ a := ⟨ne.symm, ne.symm⟩
```

Comment:
I think `ne_comm` is the more standard name (compare `eq_comm`). But admittedly there aren't a lot of examples of `*_comm` for relations.